lib/system/generic/sys.h

```diff
@@ -23,7 +23,9 @@
 #include <stdarg.h>
 #include <string.h>
 
+#ifdef __MICROBLAZE__
 #include "./@PROJECT_MACHINE@/sys.h"
+#endif
```

Comment:
I missed this one too. this breaks generic platform implementation

Reply:
just asking, why does this break the implementation? 

There is ifdef check for microblaze BSP here

Reply:
For instance, you break the "template" machine, and all machines that are not upstreamed.
Then anyone who wants to upstream their machine would have to add their own `#ifdef` statement

Reply:
Ok, we can have check only for Xilinx's platform and keep the Machine specific header.

Reply:
updated this @arnopo  .. please see latest patches